alibi_detect/saving/schemas.py

```diff
@@ -78,7 +95,7 @@ class DetectorConfig(CustomBaseModel):
 
 class ModelConfig(CustomBaseModel):
     """
-    Unresolved schema for (ML) models. Note that the model "backend" e.g. 'tensorflow', 'pytorch', 'sklearn', is set
+    Schema for (ML) models. Note that the model "backend" e.g. 'tensorflow', 'pytorch', 'sklearn', is set
```

Comment:
What's the reason for changing the description from "Unresolved schema" to just "schema"?